### ISSUE TITLE
claimed badge embedding

### ISSUE THREAD

**Issue by stenington** [mozilla/openbadges-badges#39]
After #36, this will also live on the show panel alongside #38. 

[This](https://github.com/stenington/sash) is an example of how such a thing might work, although I think maybe there's some design work to be done on what the embedded badge should look like, and some technical discussion about how to best accomplish that.

**Comment by iamjessklein** [comment]
+1 @stenington  out of curiosity, have you gotten any feedback regarding accessibility? I am wondering about legibility as well as how this translates on mobile devices? (all questions that speak to potential design implications).

**Comment by stenington** [comment]
@iamjessklein Yeah, the relative inaccessibility of the flip card has been raised, and I'm happy to ditch it for something that works for a broader audience. I'm not sure what you mean by legibility, but I guess I was referring to `sash` more as a technical reference for how it could work. Getting a designer to think through the presentation thoroughly would be cool.

**Comment by carlacasilli** [comment]
Glad that we're addressing the flip aspect. This felt complicated to me from a user perspective.

**Comment by iamjessklein** [comment]
@xmatthewx - can you take a look at the interaction design involved here?
